frontend/src/screens/wallet/Receive.tsx

```diff
@@ -116,6 +127,27 @@ export default function Receive() {
         title="Receive"
         description="Create a lightning invoice that can be paid by any bitcoin lightning wallet"
       />
+      {!!channels?.length && (
+        <>
+          {/* If all channels have less than 20% incoming capacity, show a warning */}
+          {channels?.every(
+            (channel) =>
+              channel.remoteBalance <
+              (channel.localBalance + channel.remoteBalance) * 0.2
```

Comment:
can this also check the amount? that if I enter a high amount it will also show the message?

Reply:
As in if amount is > than receiving capacity of every channel / we consider MPP and check with net receiving capacity?

Or we can also show an error if the amount entered is > receiving capacity (But is it good to restrict the user like that?)

Reply:
Done!